Create analytics.md (#21)
ScootKit/scnx-docs: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds /custom-bot/modules/administration /auto-delete.md

# Automatisches Löschen

Mit diesem Modul kannst du Nachrichten nach einer bestimmten Zeit löschen lassen, um den Kanal sauber zu halten.

<ModuleOverview moduleName="auto-delete" />

---

## Funktionen {#features}

* Lösche Nachrichten in Kanälen nach einer bestimmten Zeit automatisch
  * Optional kannst du immer eine bestimmte Anzahl an Nachrichten in einem Kanal behalten
  * Angepinnte Nachrichten werden nicht gelöscht
* Automatische Löschung von Nachrichten aus Sprachkanälen, nachdem jedes Mitglied den Kanal verlassen hat
  * Füge optional eine Verzögerung vor dem Löschen der Nachrichten hinzu

## Einrichtung {#setup}

> * Stell sicher, dass dein Bot über die Berechtigungen „Nachrichten verwalten“, „Kanal anzeigen“ und „Nachrichtenverlauf anzeigen“ für den von dir verwendeten Kanal verfügt

Reviewer: ```suggestion
* Stelle sicher, dass dein Bot über die Berechtigungen „Nachrichten verwalten“, „Kanal anzeigen“ und „Nachrichtenverlauf anzeigen“ für den von dir verwendeten Kanal verfügt
```

* Um das automatische Löschen von Nachrichten einzurichten, öffne
  die [Text-Kanäle](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cchannels) Konfigurationsdatei und

>   1. Klick auf „Neues Konfigurationselement hinzufügen“.

Reviewer: ```suggestion
  1. Klicke auf „Neues Konfigurationselement hinzufügen“.
```

  2. Wähle den Kanal aus, für den dieses Konfigurationselement gelten soll.
  3. Konfiguriere das Element wie [hier] beschrieben (#config-channels).
  4. Lade deinen Bot neu, um die Änderungen zu übernehmen.

> * Um das automatische Löschen von Nachrichten in Sprachkanälen einzurichten, nachdem jeder Benutzer den Kanal verlassen hat, öffnen

Reviewer: ```suggestion
* Um das automatische Löschen von Nachrichten in Sprachkanälen einzurichten, nachdem jeder Benutzer den Kanal verlassen hat, öffne
```

  die [Sprach-Kanäle](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) Konfigurationsdatei und

>   1. Klick auf „Neues Konfigurationselement hinzufügen“.

Reviewer: ```suggestion
  1. Klicke auf „Neues Konfigurationselement hinzufügen“.
```

  2. Wähle den Kanal aus, für den dieses Konfigurationselement gelten soll.
  3. Konfiguriere das Element wie [hier](#config-voice-channels) beschrieben.
  4. Lade deinen Bot neu, um die Änderungen zu übernehmen.

## Nutzung {#usage}

Nach Einrichten der Konfiguration arbeitet der Bot automatisch im Hintergrund, ohne dass Benutzeraktionen erforderlich ist. Um Nachrichten

> vor dem Löschen zu bewahren, pinne sie einfach an.

Reviewer: ```suggestion
vor dem Löschen zu schützen, pinne sie einfach an.
```

## Konfiguration {#configuraiton}

### Textkanäle {#config-channels}

> Mit dieser Konfigurationsdatei kannst du festlegen, von welchen Kanälen Nachrichten automatisch gelöscht werden sollen. Öffnen

Reviewer: ```suggestion
Mit dieser Konfigurationsdatei kannst du festlegen, von welchen Kanälen Nachrichten automatisch gelöscht werden sollen. Öffne
```

sie in deinem [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cchannels).

| Konfigurationsoption | Beschreibung |
|-------------|-------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- --------|
| Kanal | Dies ist der Kanal, auf den diese Konfigurationswerte angewendet werden sollen. In diesem Kanal wird die automatische Löschung durchgeführt. |

> | Timeout | Dies ist die Zeitspanne (in Minuten), nach der an diesen Kanal gesendete Nachrichten automatisch gelöscht werden sollen. Um langsamen Geräten (und langsamen Lesern) gerecht zu werden, empfehlen wir, die Zeit auf über zwei Minuten einzustellen. |

Reviewer: ```suggestion
| Timeout | Dies ist die Zeitspanne (in Minuten), nach der in diesen Kanal gesendete Nachrichten automatisch gelöscht werden sollen. Um langsamen Geräten (und langsamen Lesern) gerecht zu werden, empfehlen wir, die Zeit auf über zwei Minuten einzustellen. |
```

| Kanal beim Start leeren | Wenn diese Option aktiviert ist, löscht dein Bot automatisch jede Nachricht im Kanal (für den diese Konfigurationsoptionen gelten) beim Neustart deines Bots. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. Aufgrund einer Einschränkung seitens Discord kann der Bot nur bis zu 100 Nachrichten auf einmal löschen und keine Nachrichten löschen, die älter als 14 Tage sind. |

> | Anzahl von zu behaltenden Nachrichten | Wenn diese Option aktiviert ist, wird diese Anzahl an Nachrichten immer von der automatischen Löschung ausgenommen. Die neuesten Nachrichten bleiben immer erhalten, d. h. sobald mehr als x Nachrichten in diesen Kanal gesendet werden, löscht der Bot die älteste. Diese Zahl muss unter 50 liegen. |

Reviewer: ```suggestion
| Anzahl von zu behaltenden Nachrichten | Wenn diese Option aktiviert ist, wird diese Anzahl an Nachrichten immer von der automatischen Löschung ausgenommen. Die neuesten Nachrichten bleiben immer erhalten, d. h. sobald mehr als x Nachrichten in diesen Kanal gesendet werden, löscht der Bot die ältesten. Diese Zahl muss unter 50 liegen. |
```

Du kannst diese Datei in deinem [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) öffnen.

| Konfigurationsoption | Beschreibung |
|--------|------------ -------------------------------------------------- -------------------------------------------------- -----------------------------------------------|
| Kanal | Dies ist der Kanal, auf den diese Konfigurationswerte angewendet werden sollen. Die automatische Löschung wird in diesem Kanal durchgeführt, sobald alle Mitglieder den Kanal verlassen haben. |
| Timeout | Dies ist die Verzögerung (in Minuten) zwischen dem letzten Verlassen des Kanals durch den Benutzer und der Durchführung des Löschvorgangs. Setzen Sie diesen Wert auf Null, um sofort bei Verlassen die Löschung durchzuführen. |
| Kanal beim Start leeren | Wenn diese Option aktiviert ist, löscht dein Bot beim Neustart deines Bots jede Nachricht, wenn kein Mitglied im Sprachkanal ist. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. |

## Fehlerbehebung {#troubleshooting}

Wenn die automatische Löschung nicht durchgeführt wird, überprüfe bitte Folgendes:

> * Dein Bot benötigt folgende Rechte auf deinem Kanal: „Kanal anzeigen“, „Nachrichten verwalten“, „Nachrichtenverlauf anzeigen“, auf dem die Automatische Löschung durchgeführt werden soll.

Reviewer: ```suggestion
* Dein Bot benötigt folgende Rechte auf den Kanal, in dem die automatische Löschung durchgeführt werden soll: „Kanal anzeigen“, „Nachrichten verwalten“, „Nachrichtenverlauf anzeigen“.
```

> * Bitte stell sicher, dass die Anzahl der zu behaltenden Nachrichten unter 50 liegt.
> * Bitte stell sicher, dass neue Nachrichten an den Kanal gesendet wurden, wenn du „Beim Start löschen“ nicht aktiviert hast.
> * Bitte stell sicher, dass die Nachricht, die du löschen möchtes, nicht angepinnt ist.

Reviewer: ```suggestion
* Bitte stelle sicher, dass die Anzahl der zu behaltenden Nachrichten unter 50 liegt.
* Bitte stelle sicher, dass neue Nachrichten in den Kanal gesendet wurden, wenn du „Kanal beim Start leeren“ nicht aktiviert hast.
* Bitte stelle sicher, dass die Nachricht, die du löschen möchtest, nicht angepinnt ist.
```

> # Automatisch löschen

Reviewer 2: ```suggestion
# Automatisches Löschen
```

Mit diesem Modul kannst du Nachrichten nach einer bestimmten Zeit löschen lassen, um den Kanal sauber zu halten.

> (Nachrichten die älter als 14 Tage sind, können nicht mehr per Bot oder Automatischem, gelöscht werden)
> Die maximale Zeit die Du einstellen kannst ist also - ( 13 Tage das sind 18720 Minuten )
> Diese müssen dann Manuell gelöscht werden.

Reviewer 2: ```suggestion
```

This is not part of the original documentation document and not relevant for this module as this is designed to delete new messages? In addition, this is the introductory sentence and we'll go into this later in the document.

> Diese müssen dann Manuell gelöscht werden.

Author: fixed

<ModuleOverview moduleName="auto-delete" />

---

## Funktionen {#features}

> * Nachrichten in Kanälen nach einer bestimmten Zeit automatisch löschen
>   * Optional kannst Du immer eine bestimmte Anzahl an Nachrichten in einem Kanal behalten
>   * Kein Löschen angehefteter Nachrichten
> * Nachrichten automatisch aus einem Sprachkanal löschen, nachdem jedes Mitglied den Kanal verlassen hat
>   * Füge optional eine Verzögerung vor dem Löschen der Nachricht hinzu

Reviewer 2: ```suggestion
* Lösche Nachrichten in Kanälen nach einer bestimmten Zeit automatisch
  * Optional kannst du immer eine bestimmte Anzahl an Nachrichten in einem Kanal behalten
  * Angepinnte Nachrichten werden nicht gelöscht
* Automatische Löschung von Nachrichten aus Sprachkanälen, nachdem jedes Mitglied den Kanal verlassen hat
  * Füge optional eine Verzögerung vor dem Löschen der Nachrichten hinzu
```

>   * Füge optional eine Verzögerung vor dem Löschen der Nachricht hinzu

Author: Fixed

> ## Setup {#setup}

Reviewer 2: ```suggestion
## Einrichtung {#setup}
```

> ## Setup {#setup}

Author: Fixed

> * Stell sicher, dass Dein Bot über die Berechtigungen „Nachrichten verwalten“, „Kanal anzeigen“ und „Nachrichtenverlauf anzeigen“ für den von Dir verwendeten Kanal verfügt

Reviewer 2: ```suggestion
* Stell sicher, dass dein Bot über die Berechtigungen „Nachrichten verwalten“, „Kanal anzeigen“ und „Nachrichtenverlauf anzeigen“ für den von Dir verwendeten Kanal verfügt
```

> * Stell sicher, dass Dein Bot über die Berechtigungen „Nachrichten verwalten“, „Kanal anzeigen“ und „Nachrichtenverlauf anzeigen“ für den von Dir verwendeten Kanal verfügt

Author: Fixed

>   und du die Funktion Aktiviert hast.

Reviewer 2: ```suggestion
```
The is part of a relative sentence to the channel instead of an additional condition in the original file

>   und du die Funktion Aktiviert hast.

Author: Fixed

* Um das automatische Löschen von Nachrichten einzurichten, öffne

>   die [Text-Channels](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cchannels) Konfigurationsdatei und

Reviewer 2: ```suggestion
  die [Text-Kanäle](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cchannels) Konfigurationsdatei und
```

  1. Klick auf „Neues Konfigurationselement hinzufügen“.
  2. Wähle den Kanal aus, für den dieses Konfigurationselement gelten soll.
  3. Konfiguriere das Element wie [hier] beschrieben (#config-channels).

>   4. Lade Deinen Bot neu, um die Änderungen zu übernehmen.

Reviewer 2: ```suggestion
  4. Lade deinen Bot neu, um die Änderungen zu übernehmen.
```

* Um das automatische Löschen von Nachrichten in Sprachkanälen einzurichten, nachdem jeder Benutzer den Kanal verlassen hat, öffnen

>   die [Voice-Channels](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) Konfigurationsdatei und

Reviewer 2: ```suggestion
  die [Sprach-Kanäle](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) Konfigurationsdatei und
```

  1. Klick auf „Neues Konfigurationselement hinzufügen“.
  2. Wähle den Kanal aus, für den dieses Konfigurationselement gelten soll.

>   3. Konfiguriere das Element wie [hier] beschrieben (#config-voice-channels).

Reviewer 2: ```suggestion
  3. Konfiguriere das Element wie [hier](#config-voice-channels) beschrieben.
```
In this case, this is a link and as such direct href value is required after the label. This is in contrast to element ids, these have `{` instead of `(` and do not have a label wrapped in `[`

>   4. Lade Deinen Bot neu, um die Änderungen zu übernehmen.

Reviewer 2: ```suggestion
  4. Lade deinen Bot neu, um die Änderungen zu übernehmen.
```

## Nutzung {#usage}

> Nach einrichten der Konfiguration arbeitet der Bot automatisch im Hintergrund, ohne dass eine Benutzeraktion erforderlich ist. Um Nachrichten

Reviewer 2: ```suggestion
Nach Einrichten der Konfiguration arbeitet der Bot automatisch im Hintergrund, ohne dass Benutzeraktionen erforderlich ist. Um Nachrichten
```

> vor dem löschen zu bewahren Pinge sie einfach an.

Reviewer 2: ```suggestion
vor dem Löschen zu bewahren, pinne sie einfach an.
```

## Konfiguration {#configuraiton}

### Textkanäle {#config-channels}

Mit dieser Konfigurationsdatei kannst Du festlegen, von welchen Kanälen Nachrichten automatisch gelöscht werden sollen. Öffnen

> Dein [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cchannels).

Reviewer 2: ```suggestion
sie in deinem [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cchannels).
```

| Konfigurationsoption | Beschreibung |
|-------------|-------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- --------|

> | Kanal | Dies ist der Kanal, auf den dieser spezifische Befehlssatz angewendet werden soll. In diesem Kanal wird die automatische Löschung durchgeführt. |

Reviewer 2: ```suggestion
| Kanal | Dies ist der Kanal, auf den diese Konfigurationswerte angewendet werden sollen. In diesem Kanal wird die automatische Löschung durchgeführt. |
```

> | Zeitüberschreitung | Dies ist die Zeitspanne (in Minuten), nach der an diesen Kanal gesendete Nachrichten automatisch gelöscht werden sollen. Um langsamen Geräten (und langsamen Lesern) gerecht zu werden, empfehlen wir, die Zeit auf über zwei Minuten einzustellen. |

Reviewer 2: ```suggestion
| Timeout | Dies ist die Zeitspanne (in Minuten), nach der an diesen Kanal gesendete Nachrichten automatisch gelöscht werden sollen. Um langsamen Geräten (und langsamen Lesern) gerecht zu werden, empfehlen wir, die Zeit auf über zwei Minuten einzustellen. |
```
This value is called timeout in the dashboard

> | Beim Start bereinigen | Wenn diese Option aktiviert ist, löscht Ihr Bot automatisch jede Nachricht im Kanal (für den diese Konfigurationsoptionen gelten) beim Neustart Ihres Bots. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. Aufgrund einer Einschränkung seitens Discord kann der Bot nur bis zu 100 Nachrichten gleichzeitig löschen und keine Nachrichten, die älter als 14 Tage sind. |

Reviewer 2: ```suggestion
| Kanal beim Start leeren | Wenn diese Option aktiviert ist, löscht dein Bot automatisch jede Nachricht im Kanal (für den diese Konfigurationsoptionen gelten) beim Neustart deines Bots. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. Aufgrund einer Einschränkung seitens Discord kann der Bot nur bis zu 100 Nachrichten auf einmal löschen und keine Nachrichten löschen, die älter als 14 Tage sind. |
```
Please review automatic translations before positing them.

> | Die maximale Zeit die Du einstellen kannst ist also - 18720 Minuten, das sind 13 Tage |

Reviewer 2: ```suggestion
```
This is not part of the original document and wrongly formatted markdown

> | Anzahl der aufzubewahrenden Nachrichten | Wenn diese Option aktiviert ist, ist diese Anzahl an Nachrichten immer von der automatischen Löschung ausgenommen. Die neuesten Nachrichten bleiben immer erhalten, d. h. sobald mehr als x Nachrichten in diesen Kanal gesendet werden, löscht der Bot die älteste. Diese Zahl muss unter 50 liegen. |

Reviewer 2: ```suggestion
| Anzahl von zu behaltenden Nachrichten | Wenn diese Option aktiviert ist, wird diese Anzahl an Nachrichten immer von der automatischen Löschung ausgenommen. Die neuesten Nachrichten bleiben immer erhalten, d. h. sobald mehr als x Nachrichten in diesen Kanal gesendet werden, löscht der Bot die älteste. Diese Zahl muss unter 50 liegen. |
```

### Sprachkanäle {#config-voice-channels}

> Diese Konfigurationsdatei ermöglicht,
> von welchen Kanälen Nachrichten automatisch gelöscht werden sollen
> sobald alle Benutzer den Sprachkanal verlassen haben.

Reviewer 2: ```suggestion
Mit dieser Konfigurationsdatei kannst du einstellen, von welchen Kanälen Nachrichten automatisch gelöscht werden sollen, sobald alle Benutzer den Sprachkanal verlassen haben.
```
Please review correct syntax before publishing.

> Öffne Dein [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels).

Reviewer 2: ```suggestion
Du kannst diese Datei in deinem [Dashboard](https://scnx.app/de/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) öffnen.
```

| Konfigurationsoption | Beschreibung |
|--------|------------ -------------------------------------------------- -------------------------------------------------- -----------------------------------------------|

> | Kanal | Dies ist der Kanal, auf den dieser spezifische Befehlssatz angewendet werden soll. Die automatische Löschung wird in diesem Kanal durchgeführt, sobald alle Mitglieder den Kanal verlassen haben. |

Reviewer 2: ```suggestion
| Kanal | Dies ist der Kanal, auf den diese Konfigurationswerte angewendet werden sollen. Die automatische Löschung wird in diesem Kanal durchgeführt, sobald alle Mitglieder den Kanal verlassen haben. |
```

> | Zeitüberschreitung | Dies ist die Verzögerung (in Minuten) zwischen dem letzten Verlassen des Kanals durch den Benutzer und der Durchführung des Löschvorgangs. Setzen Sie diesen Wert auf Null, um das sofortige Löschen zu ermöglichen. |

Reviewer 2: ```suggestion
| Timeout | Dies ist die Verzögerung (in Minuten) zwischen dem letzten Verlassen des Kanals durch den Benutzer und der Durchführung des Löschvorgangs. Setzen Sie diesen Wert auf Null, um sofort bei Verlassen die Löschung durchzuführen. |
```

> | Zeitüberschreitung | Dies ist die Verzögerung (in Minuten) zwischen dem letzten Verlassen des Kanals durch den Benutzer und der Durchführung des Löschvorgangs. Setzen Sie diesen Wert auf Null, um das sofortige Löschen zu ermöglichen. |

Author: Fixed

> | Beim Start bereinigen | Wenn diese Option aktiviert ist, löscht Ihr Bot beim Neustart Ihres Bots jede Nachricht, wenn kein Mitglied im Sprachkanal vorhanden ist. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. |

Reviewer 2: ```suggestion
| Kanal beim Start leeren | Wenn diese Option aktiviert ist, löscht dein Bot beim Neustart deines Bots jede Nachricht, wenn kein Mitglied im Sprachkanal ist. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. |
```

> | Beim Start bereinigen | Wenn diese Option aktiviert ist, löscht Ihr Bot beim Neustart Ihres Bots jede Nachricht, wenn kein Mitglied im Sprachkanal vorhanden ist. Wenn deaktiviert, werden keine vorhandenen Nachrichten gelöscht. |

Author: Fixed

## Fehlerbehebung {#troubleshooting}

Wenn die automatische Löschung nicht durchgeführt wird, überprüfe bitte Folgendes:

* Dein Bot benötigt folgende Rechte auf deinem Kanal: „Kanal anzeigen“, „Nachrichten verwalten“, „Nachrichtenverlauf anzeigen“, auf dem die Automatische Löschung durchgeführt werden soll.
* Bitte stell sicher, dass die Anzahl der zu behaltenden Nachrichten unter 50 liegt.
* Bitte stell sicher, dass neue Nachrichten an den Kanal gesendet wurden, wenn Du „Beim Start löschen“ nicht aktiviert hast.

> * Bitte stell sicher, dass die Nachricht, die Du löschen möchten, nicht angepinnt ist.

Reviewer 2: ```suggestion
* Bitte stell sicher, dass die Nachricht, die du löschen möchtest, nicht angepinnt ist.
```

> * Bitte stell sicher, dass die Nachricht, die Du löschen möchten, nicht angepinnt ist.

Author: Fixed
